### DISCUSSION TITLE
Estimated Value - Currency Field missing

### DISCUSSION THREAD

**Discussion by jaggaer-fgeniola** [OP-TED/eForms-SDK#559] [Q&A] [answered]
Good evening, 
I read in a previous post that for "amount" type fields it is mandatory to enter the currency (currencyId attribute) and this is clear.
Unfortunately, however, following the SDK (1.7 at the moment) of the currency field there is no trace in the definition of the form structure (eg 4.json "OPA-27-Procedure-Currency" for "BT-27-Procedure" field) so it is not automatically generated by our software.

1) Will the field be introduced with the new SDK 1.8 version?
2) How/where can I retrieve the necessary information to correctly generate the currency field?
3) Is it possible that in the same notice there are fields with different currencies?

Thank you

**Comment by rouschr** [comment, marked as answer]
Good morning,

> Will the field be introduced with the new SDK 1.8 version?

As we received similar questions before there is ongoing work to add such fields in SDK 1.9 for the conceptual model and in the notice-types for the visual model. It means this will be supported like any other field.
A link to between the currency type field (xml attribute) and the amount field (xml element) will be provided via fields.json

> How/where can I retrieve the necessary information to correctly generate the currency field?

For pre 1.9 you can use the codelist "currency_eforms-currency.gc" to propose currencies in a select / combobox.

> Is it possible that in the same notice there are fields with different currencies?

For mixing of currency types I will let a colleague answer.
